include/triton/backend/backend_model.h

```diff
@@ -75,10 +75,8 @@ class BackendModel {
   // batch size.
   void SetMaxBatchSize(const int b) { max_batch_size_ = b; }
 
-  // Does this model support batching in the first dimension. If
-  // called before the model is completely loaded this function will
-  // return an error.
-  TRITONSERVER_Error* SupportsFirstDimBatching(bool* supports);
+  // Does this model support batching in the first dimension?
+  bool SupportsFirstDimBatching();
```

Comment:
Should keep the old interface in case it is actually used in other backend (backward compatibility), the implementation should be updated to the current form

Reply:
@GuanLuo When you return from vacation, please approve if you feel the requested changes were addressed.